client/src/components/templates/beacon-deatails.template.tsx

```diff
@@ -0,0 +1,36 @@
+import { Beacon } from "@/types/beacon";
+import ClimbingBoxLoader from "react-spinners/ClimbingBoxLoader";
+import Title from "../atoms/text/title";
+import DetailedBeacon from "../organisms/beacons/detailed-beacon";
```

Comment:
use the import alias https://nextjs.org/docs/13/app/building-your-application/configuring/absolute-imports-and-module-aliases
```suggestion
import Title from "@/components/atoms/text/title";
import DetailedBeacon from "@components/organisms/beacons/detailed-beacon";
```